src/OrchardCore/OrchardCore.ReCaptcha.Core/TagHelpers/ReCaptchaTagHelper.cs

```diff
@@ -59,7 +63,15 @@ private void ShowCaptcha(TagHelperOutput output)
             output.TagMode = TagMode.StartTagAndEndTag;
 
             var builder = new TagBuilder("script");
-            builder.Attributes.Add("src", _settings.ReCaptchaScriptUri);
+
+            var culture = _localizationService.GetDefaultCultureAsync()
+                                              .GetAwaiter()
+                                              .GetResult();
+
+            var cultureInfo = new CultureInfo(culture);
```

Comment:
```csharp
var cultureInfo = CultureInfo.GetCulture(culture);
var settingsUrl = $"{_settings.ReCaptchaScriptUri}?hl={cultureInfo.TwoLetterISOLanguageName}";
```

Reply:
I dont understand,.

Reply:
Using `CultureInfo.GetCulture` is better than new up an instance of `CultureInfo` each time

Reply:
Ok, will do.